src/stories/components/FileUpload/styles.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// FileUpload
+
+.file-upload-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.file-upload {
+  p {
+    display: inline-block;
+    margin-right: $spacing-1;
+  }
+}
+
+.upload-icon {
+  p {
+    margin-bottom: 0;
+    display: flex;
+    justify-content: center;
+  }
+  img {
+    margin-left: calc(#{$spacing-1} / 3);
+  }
+}
+
+input[type="file"] {
```

Comment:
You need to use a class selector. Otherwise, every single file input would be affected: `file-upload__input`.